contracts/test_api/test_account.cpp

```diff
@@ -0,0 +1,27 @@
+#include <eoslib/account.hpp>
+#include <eoslib/eos.hpp>
+#include <eoslib/system.h>
+
+#include "test_api.hpp"
+
+/// eos with 8 digits of precision
+#define EOS_SYMBOL  (int64_t(4) | (uint64_t('E') << 8) | (uint64_t('O') << 16) | (uint64_t('S') << 24))
```

Comment:
This is defined in asset.hpp, can't you just include it? BTW the comment is wrong.